blog/_posts/2018-06-13-accessible-scala.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog-detail
+post-type: blog
+by: Guillaume Massé
+title: Introducing Accessible Scala
+---
+
+  Have you found yourself where you had to verbally describe code? For example, while pair-programming
+or talking about work during lunch. Verbally describing code is not an easy task, fortunately for you the
+Scala Center is working on this. [SCP-016]: Accessible Scala, was submitted to provide industry-leading support
+for blind and partially-sighted developers. It's also an opportunity to formalize a spoken Scala. For example,
+how do you pronounce: `Map[K, V]`? Perhaps `Map of K V`, or maybe `Map taking K and V`.
+
+  Reading Scala out loud make some of its syntactic elements less intimidating for beginners. When we read the
+following expression: `class S[+T]` as `class S parameterized with covariant T`, we don't need to mentally
+associate the syntax `+T` with its concept `co-variant`.
+
+  One of the goals of the proposal is to remove the noise introduced by various delimiters. Notice how the type
+parameter delimiters: `[` and `]` are absent from the verbal description. However, with more complex expressions, 
+this can make the audible form ambiguous, for example, with nested types: `Either[Wobble[T], Option[Wobble[S]]]`
+becomes: `Either applied to types Wobble of T, Option of Wobble of S`.
```

Comment:
Again, "applied to" is not something everybody agrees with. It's not appropriate to act as if the Scala Center has a made a decision on how to pronounce these things. This is for blind developers.